src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ArrayInitHandler.java

```diff
@@ -99,7 +99,9 @@ protected IndentLevel getChildrenExpectedIndent() {
         final int lcurlyPos = expandedTabsColumnNo(getLeftCurly());
         final int firstChildPos =
             getNextFirstNonBlankOnLineAfter(firstLine, lcurlyPos);
-        if (firstChildPos >= 0) {
+        final int notExist = -1;
```

Comment:
Please torn it into a constant field and use it here and in the `getNextFirstNonBlankOnLineAfter` method.